frontend/src/stories/SectionHeaders.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
+import { moduleMetadata } from '@storybook/angular';
+
+# Section Headers
+
+> Example: Section header example
+
+The section header is used to separate different sections of a page or a view, or different grouping of similar functionality. 
+
+In the Files tab for example, the section header separates the Attachments section and the the different file storages that may be available to a work package.
+
+## Structure and Options
+
+Section headers are at their most basic form a label (in all-caps) with a line underneath. 
+
+> Example of a basic section header
+
+However, they can be extended with a few additional features:
+
+**Left icon**
+This icon can be added to provide context. In a files list, for example, the left icon is used to indicate to which storage provider (for example, Nextcloud) that file storage is linked.
+
+In some parts of the application, this icon is also used to indicate open/closed state when the section header functions as an accordian (see *Behaviour*). 
+
+*Note: Since this element is always all-caps, the baseline of the font requires the icon to be raised by a few pixels. See* Margins and Spacing *for more details.*
+
+**Right icon**
+The icon can be placed to the right of the text too, if required. This should be used sparingly.
```

Comment:
Maybe indicate that this icon can be a link if needed.

Reply:
Added.